book/P1C5_eha.qmd

```diff
@@ -12,159 +12,456 @@ abstract: TODO (150-200 WORDS)
+Transition probabilities of a multi-state process are often summarized in a quadratic matrix of transition probabilities
+<!--  -->
+$$
+\mathbf{P}(\zeta, \tau):=
+\begin{pmatrix}
+  P_{00}(\zeta, \tau) & \cdots & P_{0q}(\zeta, \tau)\\
+  \vdots & \ddots & \vdots\\
+  P_{q0}(\zeta, \tau) & \cdots & P_{qq}(\zeta, \tau)\\
+\end{pmatrix},
+$$ {#eq-ms-trans-prob-mat}
+<!--  -->
+where rows indicate starting states and columns indicate end states.
+Some of the elements of $\mathbf{P}$ might be zero or one depending on the specific process, presence of absorbing states and possible pathways between states.
+As subjects can only be in one of the $q+1$ states at $\tau$, rows have to sum up to 1:
+<!--  -->
+$$
+\sum_{e =0}^q P_{\ell e} = 1,\forall \ell \in \{0, \ldots,q\}.
+$$ {#eq-ms-trans-prob-row-sums}
+<!--  -->
 
-$$\mathbf{P}(\zeta, \tau) :=
+For illustration, consider the illness-death model (left panel of @fig-multi-state).
+In this example, back-transitions are not possible, therefore the lower triangle of the matrix is filled with zeros and $P_{22}(\zeta, \tau)\equiv 1, \forall\ \zeta < \tau$ by virtue of being an absorbing state. The transition matrix is thus given as
+<!--  -->
+$$
+\mathbf{P}(\zeta, \tau) =
   \begin{pmatrix}
   P_{00}(\zeta, \tau) & P_{01}(\zeta, \tau) & P_{02}(\zeta, \tau)\\
-  \bullet & P_{11}(\zeta, \tau) & P_{12}(\zeta, \tau)\\
-  \bullet & \bullet & P_{22}(\zeta, \tau)\\
+  0 & P_{11}(\zeta, \tau) & P_{12}(\zeta, \tau)\\
+  0 & 0 & 1\\
+  \end{pmatrix}
+$$ {#eq-ms-trans-prob-mat-illness-death}
+<!--  -->
+So far, this is not very useful for estimation.
+Thus, for now, assume that data is collected in discrete time, that is $\zeta, \tau \in \{0, 1, 2,\ldots\}, \zeta < \tau$ and transitions only occur at these discrete time points and not in between.
+Say we are interested in transition probability $P_{02}(4, 6)$, that is the probability to transition from state $0$ to state $2$ between time points $\zeta=4$ and $\tau=6$, given we are in state $0$ at time $\zeta = 4$.
+
+This is possible in three ways:
+
+- stay in state $0$ at time $5$, transition to state $2$ at time $6$,
+transition to state $1$ at time $5$, transition to state $2$ at time $6$, or
+- transition to state $2$ directly at time $5$.
+
+Thus, $P_{02}(4,6) = p_{00}(5)p_{02}(6) + p_{01}(6)p_{12}(6) + p_{02}(5)$, where $p_{\ell e}(\tau)=P(E(\tau)=e|E(\tau -1) = \ell)$ are the probabilities for transitions $\ell \rightarrow e$ between two discrete time points.
+Thus, in discrete time, the matrix of transition probabilities can be represented as a finite matrix product (not element wise product)
+
+$$
+\mathbf{P}(\zeta, \tau) = \prod_{j =\zeta+1}^{\tau}
+\begin{pmatrix}
+  p_{00}(j) & p_{01}(j) & p_{02}(j)\\
+  0 & p_{11}(j) & p_{12}(j)\\
+  0 & 0 & 1\\
   \end{pmatrix}.
+$$ {#eq-ms-trans-prob-ill-death-discrete}
```

Comment:
Yeah so this goes from being concrete to abstract then concrete again. If this was higher, next to P, then it would be easier to show the relation between P_xy and p_xy. *then* move on to the example